src/objects/Trigger_Handler__c.object

```diff
@@ -89,6 +89,16 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Exclude_For_These_Usernames__c</fullName>
```

Comment:
@smarovich Are you ok with the inline help text and description values for this new field?